From d2cd2ba6d23662640cf3661b7210450f13bc252f Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Fri, 11 Jul 2025 10:32:38 +0200
Subject: [PATCH] bytedevkit-am62x: Set byteDEVKIT2.2 as new default

byteDEVKIT2 Rev 2.0 is the new default.

byteDEVKIT2 Rev 1.0 can be selected by executing the following commands
in the u-boot console:
  setenv dtbfile k3-am625-bytedevkit.dtb
  saveenv
  reset

Signed-off-by: Daniel Ammann <daniel.ammann@bytesatwork.ch>
---
 include/configs/am62x_bytedevkit.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/am62x_bytedevkit.h b/include/configs/am62x_bytedevkit.h
index c54cf296ab5..51d2ce25114 100644
--- a/include/configs/am62x_bytedevkit.h
+++ b/include/configs/am62x_bytedevkit.h
@@ -21,7 +21,7 @@
 
 #define CFG_EXTRA_ENV_SETTINGS \
 	"console=ttyS2,115200n8\0" \
-	"dtbfile=" CONFIG_DEFAULT_DEVICE_TREE ".dtb\0" \
+	"dtbfile=k3-am625-bytedevkit-2-2.dtb\0" \
 	"kernelfile=Image\0" \
 	"loadaddr=0x82000000\0" \
 	"dtbaddr=0x88000000\0" \
